plugins/netskope/help.md

```diff
@@ -0,0 +1,556 @@
+# Description
+
+Update a File Hash List using Netskope
+
+# Key Features
+
+Identify key features of plugin.
```

Comment:
Add some key features (list of actions) here

Reply:
Same as above :smiley: